### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by GzimAsani**
Millstone 4:
- We started by telling the user the required installations before starting the game
- We explained the logic of the game (How to play)
- Explained on how to download the game and play

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+>The purpose of this project was to demonstrate our learned skills on Ruby programming. Here we developed the Tic-Tac-Toe game.
+
+<img src="screenshot/screen.png">
+
+In this project we created an executable file on main.rb that allows the users to play the game through terminal
+
+## How To Play
+
+"Tic Tac Toe" is 2 player game which includes 9 squares each of them represented with a number. Each player will be represented with a symbol ("X" or "O") and their respective symbols will overwrite the square numbers. The starter of the game will be the first player.
+In case none of the players manages to put three of their symbols in a straight line or in a diagonal line the game will be end on draw.
```
Comment (euqueme) [inline README.md]: - [ ] Check this section with Grammarly some improvements have to be made for instance change "is 2 player" to "is a 2 player"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+In order to play it locally follow the next steps
+
+### Prerequisites 
+
+Ruby installed localy 
+
+### Install
+
+1. Clone the repo into your machine
+    $ git@github.com:GzimAsani/Tic-Tac-Toe-Game.git
```
Comment (euqueme) [inline README.md]: - [ ] They don't need to clone the repo they can also just download it, let's add that option to this step, also use the insert code tag `< >`  for the git command

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+2. Navigate to the downloaded file and click bin folder than main.rb
+
+3. Type in bin/main.rb in the terminal.
```
Comment (euqueme) [inline README.md]: - [ ]  Specify this step is how you start the game. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request Reviewed by Maria Eugenia Queme P. (TSE)

### Hello @GzimAsani and @CornerstoneII :balloon: :lollipop: 

### Thank you for your submission :handshake: :handshake: , you've done a good job with your repository and Readme file :ok_hand: :medal_sports: the changes to make are small, and I can tell from your code that you've worked hard in this project so claps to your for all your efforts :clap: :clap: :clap:  let's work on the following changes :construction: :construction_worker: :construction_worker: 

### Behaviour

- [ ] The game is breaking when the first players' name is input, let's fix it, the game should run smoothly at this point and it shouldn't break with any kind of input
image

### Milestone 4 (Readme file)

- [ ] I've left comments directly in your Readme file for you to address 

### That's it, team! :raised_hands: 

### Please :pray: do the changes :technologist: :technologist: and submit back for a code review when you are done :memo: 

### Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+### Prerequisites 
+
+Ruby installed localy 
+
+### Install
+
+1. Clone the repo into your machine
+    $ git@github.com:GzimAsani/Tic-Tac-Toe-Game.git
+
+2. Navigate to the downloaded file and click bin folder than main.rb
```
Comment (maelfosso) [inline README.md]: Instead of writing that, use command
`cd Tic-Tac-Toe-Game`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+In order to play it locally follow the next steps
+
+### Prerequisites 
+
+Ruby installed localy 
+
+### Install
+
+1. Clone the repo into your machine
+    $ git@github.com:GzimAsani/Tic-Tac-Toe-Game.git
```
Comment (maelfosso) [inline README.md]: Instead of SSH way use HTTPs one
`git clone https://github.com/...`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+2. Navigate to the downloaded file and click bin folder than main.rb
+
+3. Type in bin/main.rb in the terminal.
```
Comment (maelfosso) [inline README.md]: It's rather `./bin/main.rb`. Add also the other method `ruby bin/main.rb`

**Review by maelfosso — CHANGES_REQUESTED** [review]
# CHanges required 🔴 

Hello,

Thank you for implementing the suggested changes
However, there is some issue. 
Kindly read the comments left below to know more about the issues.

Once solved, submit a new code review request.
Thanks

Mael FOSSO

File: README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+In order to play it locally follow the next steps
+
+### Prerequisites 
+
+Ruby installed localy 
+
+### Install
+
+1. Clone the repo into your machine
+    $ https://github.com/GzimAsani/Tic-Tac-Toe-Game.git
```
Comment (kobiyoyo) [inline README.md:30]: it should be written like this `clone https://github.com/GzimAsani/Tic-Tac-Toe-Game.git`

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello Team  ,  you have done a brilliant job with this project. :clap:
#### You can merge  once done!!!

**Pull request merged**